docs/source/en/api/pipelines/unclip.md

```diff
@@ -7,9 +7,9 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 specific language governing permissions and limitations under the License.
 -->
 
-# UnCLIP
+# unCLIP
```

Comment:
Can you also align the title in the `toctree` with the title here?

Reply:
Done here: https://github.com/huggingface/diffusers/pull/5587/files#diff-70bf28e04d0044ad94e999d388a8c73056b57a15696b29d8773efa01ed5917e4